google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -442,4 +453,26 @@ public StatementResult statementShowRPCPriority() {
   public StatementResult statementShowTransactionIsolationLevel() {
     return resultSet("transaction_isolation", "serializable", SHOW_TRANSACTION_ISOLATION_LEVEL);
   }
+
+  @Override
+  public ResultSet statementExplain(String sql) {
+
+    sql = sql.trim();
+
+    String[] arr = sql.split(" +", 2);
+
+    String option = arr[0];
+    String statementToBeExplained = arr[1];
```

Comment:
Check to ensure that we did not just receive something like `explain foo`, which would cause an `ArrayOutOfBoundsException` here. Also add tests for that kind of corner cases.